node/graphql/index.ts

```diff
@@ -126,15 +126,18 @@ export const resolvers = {
       const {
         data,
         data: { paging },
-      } = result
+      } = result ?? { data: { items: [] } }
 
       const pickuppoints = data.items ? data : { items: data }
 
       if (paging?.pages > 1) {
         let i = 2
         const results = [] as any
 
-        while (i <= paging.pages) {
+        // API will return errors starting at ?page=100
+        const limitPagesTo99 = paging.pages > 99 ? 99 : paging.pages
+
+        while (i <= limitPagesTo99) {
           results.push(hub.getStores({ ...param, page: i }))
```

Comment:
This If, while can be done more elegantly with a for loop
Something like
For (let i = 2; i <= pageLim; i++)

Reply:
Done!